docs/events/jarsigning.md

```diff
@@ -0,0 +1,83 @@
+Signing a JAR
+=============
+
+Forge allows Mods to sign their jar files with a signature. However, these signatures are not used
```

Comment:
- This is nothing Forge specific. Jar-Signing is a Java feature.
- "are not used as" => "are not designed to be used as"